python/apps/in-mem-dynamic.py

```diff
@@ -18,7 +18,25 @@ def insert_and_search(
     num_insert_threads,
     num_search_threads,
     gt_file,
-):
+) -> dict[str, float]:
```

Comment:
annoyingly, this annotation may or may not be valid wrt the version of python you're using.  I think it became standard in 3.10 but in 3.9 it might still be `from typing import Dict`.  I actually can't remember the specific version but it would be an issue - but since we're the ones using this for testing purposes, we can probably just ignore that little bit entirely

Reply:
nevermind, it was 3.9 that changed it to builtins.dict and deprecated the typing.Dict